### ISSUE TITLE
Move ability effects to ActiveEffects

### ISSUE THREAD

**Issue by bithir** [pwatson100/symbaroum#612]
Currently we have to hodge podge together some of the abilities to make them work well in the system.  With v11, active effects can now be placed on items and not transferred, which opens up a lot of options for us.
As such, I would suggest improvements starting with existing berserk & blessed shield and replace them with ActiveEffects.  Once that is completed, we can extend this functionality to enable poison, elemental effects, etc. etc.

**Comment by bithir** [comment]
```
actor.createEmbeddedDocuments("ActiveEffect", [{
  name: "Name of Effect",
  icon: "your icon here",
  description: "Description of effect.",
  origin: "uuid of where it 'originates', optional",
  changes: [],
  statuses: []
}]);
```

**Comment by bithir** [comment]
https://foundryvtt.com/article/active-effects/

**Comment by bithir** [comment]
Fixed in 4.4
